ops/docker-compose-omgx-services.yml

```diff
@@ -30,3 +30,43 @@ services:
     ports:
         # expose the service to the host for getting the contract addrs
       - ${WALLET_DEPLOYER_PORT:-8078}:8079
+
+  bl-wl:
+    depends_on:
+      - l1_chain
+      - deployer
+      - wallet_deployer
+      - l2geth
+    image: omgx/bl-wl
+    build:
+      context: ..
+      dockerfile: ./ops/docker/Dockerfile.bl-wl
+    environment:
+        WALLET_URL: http://wallet_deployer:8079/addresses.json
+        BL_WL_PORT: 9000
+    ports:
+      - 9000:9000
+
+  message-relayer-fast:
+    depends_on:
+      - l1_chain
+      - deployer
+      - wallet_deployer
+      - l2geth
+      - bl-wl
+    image: omgx/message-relayer-fast
+    build:
+      context: ..
+      dockerfile: ./ops/docker/Dockerfile.message-relayer-fast
+    environment:
+        L1_NODE_WEB3_URL: http://l1_chain:8545
+        L2_NODE_WEB3_URL: http://l2geth:8545
+        L1_WALLET_KEY: "0xde9be858da4a475276426320d5e9262ecfc3ba460bfac56360bfa6c4c28b4ee0"
+        L1_TARGET: 0x0
```

Comment:
`L1_TARGET` is replaced by `WHITELIST_ENDPOINT`. Please remove it.

Reply:
Cool!